app/models/pseud.rb

```diff
@@ -214,7 +212,7 @@ def unposted_works
 
   # Produces a byline that indicates the user's name if pseud is not unique
   def byline
-    (name != user_name) ? name + " (" + user_name + ")" : name
+    (name != user_name) ? "#{name} (#{user_name})" : name
```

Comment:
I vote we ignore Hound here. At least for me, parentheses made it more readable.

Reply:
Should we disable Style/TernaryParentheses in `.rubocop.yml`?

Reply:
Yeah, let's. I don't think we use them very often, only when we see a benefit in them.

Reply:
Should I add it in this PR?

(Also, as an alternative to disabling it, it can apparently be set to [`require_parentheses_when_complex`](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/TernaryParentheses), which might cover this case.)

Reply:
Please do include the change for `.rubocop.yml` (either alternative), thanks!